### PR TITLE
[IMP] point_of_sale, pos_restaurant: add images on demo categories

### PR DESCRIPTION
**Pull request by adgu-odoo**
In this commit, we add images on demo categories to make the demo more realistic but also change a bit the design of the category selector as it was not adapted to display images.

Old without image:
image

Old with images:
image

New without images:
image

New with images:
image

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/static/src/app/screens/product_screen/product_list/control_panel/control_panel.scss [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-    & img {
-        max-height: 100px;
-        max-width:  120px;
-    }
-}
+//     & img {
+//         max-height: 100px;
+//         max-width:  120px;
+//     }
+// }
```
Comment (caburj) [inline addons/point_of_sale/static/src/app/screens/product_screen/product_list/control_panel/control_panel.scss:12]: decide whether to keep or not.

**Comment by davidmonnom** [comment]
Hello, in runbot on a classic PoS category images arn't shown:

image

is it normal ?

**Comment by davidmonnom** [comment]
Apparently, the z-index of the category bar is higher than that of the popup window background.

image

(same for loyalty point if you can correct this at the same time, increase z-index of popup background should be good)

**Comment by adgu-odoo** [comment]
> Hello, in runbot on a classic PoS category images arn't shown:
> 
> image
> 
> is it normal ?

I think it is because I made a test on the runbot yesterday and I removed them during this test. If create a db in local you will see the images

**Comment by davidmonnom** [comment]
@robodoo r+

**Comment by davidmonnom** [comment]
@robodoo r+

**Pull request closed without merging**